src/backend/LDW.WebAPI/Controllers/v1/AccountController.cs

```diff
@@ -47,8 +46,7 @@ public async Task<IActionResult> Registration([FromBody] RegisterUserApiModel mo
             var userModel = new UserModel
             {
                 Email = model.Email,
-                UserName = model.Email,
-                EmailConfirmed = false
+                UserName = model.Email
```

Comment:
username == email, it should be so?

Reply:
We should set some default user name(in case user only registers and does not specify user name in edit profile page). I decided to set it to email by default, but it can be a subject to discuss.